Dockerfile.preflight

```diff
@@ -0,0 +1,13 @@
+# This builds a preflight playbook image for use with Docker.
+# Look at playbooks/byo/preflight/README.md for more information.
+# docker build -t docker.io/sosiouxme/ocp_preflight_playbook -f Dockerfile.preflight .
+
+FROM docker.io/aweiteka/playbook2image
```

Comment:
I questioned the need myself, aweiteka thought it would be good to have a standard way of doing these playbook images, and I didn't think a lot more about it. It does provide a little bit of logic in the entrypoint for handling some parameters, and that could grow more complex, so I'm kind of inclined to keep it. But I can see it either way.